modules/ROOT/pages/ibm-ctg-connector.adoc

```diff
@@ -556,12 +572,6 @@ image::ibm-ctg-use-case-5.png[]
 . Click the Select button to save the configuration.
 . Perform the same steps to load metadata for the `ChannelRequest` type.
 
-== Connector Performance
-
-To define the pooling profile for the connector manually, access the Pooling Profile tab in the applicable global element for the connector.
-
-For background information on pooling, see xref:tuning-performance.adoc[Tuning Performance].
-
 == See Also
 
 * Access the xref:release-notes::connector/ibm-ctg-connector-release-notes.adoc[IBM CICS Transaction Gateway Connector Release Notes]
```

Comment:
These are called IBM CTG Connector Release Notes above; which is the actual title?